crates/ty_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -688,6 +688,30 @@ class C[T]:
 c: Callable[[int], C[int]] = C[int]
 ```
 
+### Generic callables and TypeVar identity
```

Comment:
This test looks great. I think ideally we'd have a version of it using PEP 695 typevars also?

Reply:
No worries